src/Protocol/Protocol_1_9.cpp

```diff
@@ -104,6 +104,78 @@ void cProtocol_1_9_0::SendAttachEntity(const cEntity & a_Entity, const cEntity &
 
 
 
+void cProtocol_1_9_0::SendBossBar(const cBossBar & a_BossBar, cBossBar::eOperation a_Operation)
+{
+	ASSERT(m_State == 3);  // In game mode?
+	cPacketizer Pkt(*this, pktBossBar);
+	// TODO: Bad way to write a UUID, and it's not a true UUID, but this is functional for now.
+	Pkt.WriteBEUInt64(0);
+	Pkt.WriteBEUInt64(a_BossBar.GetUniqueID());
+	Pkt.WriteVarInt32(static_cast<unsigned int>(a_Operation));
+	LOG(a_BossBar.GetTitle().CreateJsonString());
+	LOG(a_BossBar.GetJsonTitle());
```

Comment:
🪵

Reply:
this emoji is apparently too new, it should be of a wooden log :p

Reply:
I see it :o 

Yea I had some fun with composite chat crashing the server